advanced/am-pattern-2/confs/am-pub-store-tm/instance-1/repository/conf/deployment.toml

```diff
@@ -0,0 +1,87 @@
+[server]
+hostname = "$env{HOST_NAME}"
+node_ip = "$env{NODE_IP}"
+mode = "single" #single or ha
+base_path = "${carbon.protocol}://${carbon.host}:${carbon.management.port}"
+server_role = "default"
+
+[super_admin]
+username = "admin"
+password = "admin"
+create_admin_account = true
+
+[user_store]
+type = "database_unique_id"
+
+[database.apim_db]
+type = "mysql"
+url = "jdbc:mysql://wso2am-mysql-db-service:3306/WSO2AM_DB?useSSL=false&amp;autoReconnect=true&amp;requireSSL=false&amp;verifyServerCertificate=false"
+username = "wso2carbon"
+password = "wso2carbon"
+driver = "com.mysql.cj.jdbc.Driver"
+
+[database.shared_db]
+type = "mysql"
+url = "jdbc:mysql://wso2am-mysql-db-service:3306/WSO2AM_SHARED_DB?useSSL=false&amp;autoReconnect=true&amp;requireSSL=false&amp;verifyServerCertificate=false"
+username = "wso2carbon"
+password = "wso2carbon"
+driver = "com.mysql.cj.jdbc.Driver"
+
+[keystore.tls]
+file_name =  "wso2carbon.jks"
+type =  "JKS"
+password =  "wso2carbon"
+alias =  "wso2carbon"
+key_password =  "wso2carbon"
+
+[[apim.gateway.environment]]
+name = "Production and Sandbox"
+type = "hybrid"
+display_in_api_console = true
+description = "This is a hybrid gateway that handles both production and sandbox token traffic."
+show_as_token_endpoint_url = true
+service_url = "https://wso2am-pattern-2-gateway-service:${mgt.transport.https.port}/services/"
+username= "${admin.username}"
+password= "${admin.password}"
+ws_endpoint = "ws://wso2am-pattern-2-gateway-service:9099"
+wss_endpoint = "wss://wso2am-pattern-2-gateway-service:8099"
+http_endpoint = "http://{{ .Release.Name }}-gateway"
+https_endpoint = "https://{{ .Release.Name }}-gateway"
+
+[apim.analytics]
+enable = true
+store_api_url = "https://wso2am-pattern-2-am-analytics-worker-service:7444"
+
+[[apim.analytics.url_group]]
+analytics_url =["tcp://wso2am-pattern-2-analytics-worker-1-service:7612","tcp://wso2am-pattern-2-analytics-worker-2-service:7612"]
+analytics_auth_url =["ssl://wso2am-pattern-2-analytics-worker-1-service:7712","ssl://wso2am-pattern-2-analytics-worker-2-service:7712"]
+type = "failover"
+
+[apim.devportal]
+url = "https://{{ .Release.Name }}-am/devportal"
+
+[transport.https.properties]
+proxyPort = 443
+
+[apim.cors]
+allow_origins = "*"
+allow_methods = ["GET","PUT","POST","DELETE","PATCH","OPTIONS"]
+allow_headers = ["authorization","Access-Control-Allow-Origin","Content-Type","SOAPAction"]
+allow_credentials = false
+
+[apim.throttling]
+enable_data_publishing = true
+#enable_policy_deploy = true
+enable_blacklist_condition = false
+
+[[apim.throttling.url_group]]
+traffic_manager_urls = ["tcp://localhost:9611","tcp://wso2am-pattern-2-pub-store-tm-2-service:9611"]
```

Comment:
@Chathuru @pubudu538 shouldn't these be configured in multiple `apim.throttling.url_group` as in `am-pattern-1`?

Reply:
Fix in https://github.com/wso2/kubernetes-apim/pull/344/commits/33f03369f79ab2dbfe408b29cd8aaea47552b6c9 commit